src/Microsoft.Diagnostics.Monitoring.StartupHook/Exceptions/Eventing/ExceptionsEventSource.cs

```diff
@@ -79,16 +79,18 @@ public void ClassDescription(
             ulong ModuleId,
             uint Token,
             uint Flags,
+            uint StackTraceHidden,
             ulong[] TypeArgs)
         {
-            Span<EventData> data = stackalloc EventData[5];
+            Span<EventData> data = stackalloc EventData[6];
             Span<byte> typeArgsSpan = stackalloc byte[GetArrayDataSize(TypeArgs)];
             FillArrayData(typeArgsSpan, TypeArgs);
 
             SetValue(ref data[NameIdentificationEvents.ClassDescPayloads.ClassId], ClassId);
             SetValue(ref data[NameIdentificationEvents.ClassDescPayloads.ModuleId], ModuleId);
             SetValue(ref data[NameIdentificationEvents.ClassDescPayloads.Token], Token);
             SetValue(ref data[NameIdentificationEvents.ClassDescPayloads.Flags], Flags);
+            SetValue(ref data[NameIdentificationEvents.ClassDescPayloads.StackTraceHidden], StackTraceHidden);
```

Comment:
While we are not trying to support any kind of version compatibility for the event data between .NET Monitor and the startup hook event sources, I would prefer that we would append new event data to the existing definition rather than inserting it in the middle.

Reply:
This was an intentional decision -- Since like you said we're not trying to support any kind of version compatibility I'm of the opinion that we shouldn't worry about it yet and should be free to re-order events if it makes sense.

Fixed-sized payloads should generally be specified before variable-length payloads (such as strings / arrays) for improved & faster deserialization. The only exception being if we cared about backwards compatibility and utilized the `Version` attribute in which case it makes sense to either append or create a new event name / event source name altogether.